airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -138,7 +138,10 @@ class BigQueryPandasConnector(GbqConnector):
     without forcing a three legged OAuth connection. Instead, we can inject
     service account credentials into the binding.
     """
-    def __init__(self, project_id, service, reauth=False, verbose=False, dialect='legacy'):
+    def __init__(self, project_id, service, reauth=False, verbose=False, dialect='legacy', **kwargs):
```

Comment:
Maybe we can put a comment pointing to the relevant JIRA issue, so we can check when it is fixed.

And probably we should make this commit a different PR, so it is easy to reset the patches separately when their are fixed.

Reply:
I'm closing this PR as this has been split into two PRs

[Fix Attribute Error PR](https://github.com/wizeline/incubator-airflow/pull/4)

and  

[Workaround for Pandas GBQ](https://github.com/wizeline/incubator-airflow/pull/5)